src/main/java/com/booksaw/betterTeams/events/MCTeamManagement.java

```diff
@@ -42,6 +43,10 @@ public void displayBelowNameForAll() {
 	}
 
 	public void displayBelowName(Player player) {
+
+		if (FoliaUtils.isFolia())
+			return; // Scoreboard stuff doesn't work on folia :(
```

Comment:
@booksaw : breaking?
@maiminhdung : any way to make it work?

Reply:
If none of the McTeamManagement class can function as expected with Folia due to it not currently having support. 
Could we disable the class entirely by in the Main.java with some nice logging? Simply returning like this does would theoretically work, but will make it more of a challenge to maintain and understand for people who are not familiar with Folia (i.e. me)

https://github.com/booksaw/BetterTeams/blob/b16315ece281aa83b70715aacfc5d7fe00023070/src/main/java/com/booksaw/betterTeams/Main.java#L341